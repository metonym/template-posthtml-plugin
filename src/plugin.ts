import { PostHTML } from 'posthtml';

function plugin() {
  return function posthtmlPlugin(tree: PostHTML.Node) {
    return tree;
  };
}

export { plugin };
