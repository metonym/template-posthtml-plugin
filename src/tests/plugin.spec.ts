import posthtml from 'posthtml';
import plugin from '../';

describe('posthtml-plugin', () => {
  it('matches the snapshot', () => {
    posthtml()
      .use(plugin())
      .process(`<body></body>`)
      .then((result: { html: string }) => {
        expect(result.html).toMatchSnapshot();
      });
  });
});
