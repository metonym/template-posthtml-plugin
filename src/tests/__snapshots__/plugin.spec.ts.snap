// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`posthtml-plugin matches the snapshot 1`] = `"<body></body>"`;
